_users/cn/setup-environment.md

```diff
@@ -31,19 +31,19 @@ last_modified_at: 2017-09-03T10:01:43-04:00
    
    <div class="tab-content">
      <div id="windows" class="tab-pane active" markdown="1">
-   1. 下载[服务注册中心可执行文件压缩包](https://github.com/apache/incubator-servicecomb-service-center/releases/download/0.5.0/service-center-0.5.0-windows-amd64.zip)
+   1. 下载[服务注册中心可执行文件压缩包](http://www-eu.apache.org/dist/incubator/servicecomb/incubator-servicecomb-service-center/1.0.0-m1/apache-servicecomb-incubating-service-center-1.0.0-m1-windows-amd64.tar.gz)
```

Comment:
Please use  the below link http://apache.org/dyn/closer.cgi/incubator/servicecomb/incubator-servicecomb-service-center/1.0.0-m1/apache-servicecomb-incubating-service-center-1.0.0-m1-windows-amd64.tar.gz

Reply:
Done